types/react-calendar-heatmap/tsconfig.json

```diff
@@ -16,6 +16,7 @@
         "types": [],
         "noEmit": true,
         "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true,
```

Comment:
This config option should not be present in a types package, see: https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports

Reply:
I removed it! Thanks